src/datasources/asset-calibration/AssetCalibrationDataSource.ts

```diff
@@ -74,6 +96,8 @@ export class AssetCalibrationDataSource extends DataSourceBase<AssetCalibrationQ
       [AssetCalibrationForecastGroupByType.Day]: this.formatDateForDay,
       [AssetCalibrationForecastGroupByType.Week]: this.formatDateForWeek,
       [AssetCalibrationForecastGroupByType.Month]: this.formatDateForMonth,
+      [AssetCalibrationForecastGroupByType.Location]: (v: string) => v,
+      [AssetCalibrationForecastGroupByType.Model]: (v: string) => v,
```

Comment:
It doesn't look like these will ever be called because line 103 has `formatFunctionMap[timeGrouping!]`

Reply:
Will remove these